uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -427,6 +434,46 @@ private static function getNotificationReceiversIDs($typeId) {
   }
 
   /**
+   * Checks, based on the $params array, if the Public Holiday Leave Requests
```

Comment:
well , I Am not an English wizard :tophat:  but it sounds more correct to me without the colon .. but maybe I am wrong